### PR TITLE
Bringing up dashboard webapp

### PR DESCRIPTION
**Pull request by kartikaditya**
1. Fixed some code in the ControllerModule to bring up the webapp
2. Added relevant config to yml file
3. Added simple test case to bring up the service

### CODE REVIEW

**Comment by kartikaditya** [comment]
@yogeshnachnani @shyamakirala - Kindly review

File: runtime/src/main/java/com/flipkart/flux/config/FileLocator.java [hunk 1]
```diff
@@ -42,9 +43,11 @@
 	 */
 	private static final Logger LOGGER = LogManager.getLogger(FileLocator.class);
 	
-	/** The root folder where Flux runtime is deployed*/
-	@Inject
-	private static String deploymentRoot;
+	/** The config root folder of Flux */
+	private static final String DEPLOYED_CONFIG_ROOT;
```
Comment (yogeshnachnani) [inline runtime/src/main/java/com/flipkart/flux/config/FileLocator.java]: Static?
We wanted it to be a configuration, hence added the Inject annotation

File: runtime/src/test/java/com/flipkart/flux/controller/DashboardWebAppTest.java [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+ * @author kartik.bommepally
+ */
+public class DashboardWebAppTest {
+
+    private static Injector injector;
+    private static Server server;
+    private static int port;
+
+    @BeforeClass
+    public static void beforeClass() throws Exception {
```
Comment (yogeshnachnani) [inline runtime/src/test/java/com/flipkart/flux/controller/DashboardWebAppTest.java]: Could you please leave a TODO for me to replace this with the guiceJunit4Runner? I have added it in my branch, will pull it into this one as well

**Comment by yogeshnachnani** [comment]
looks good - we can change everything from RunTimeConstants into configuration whenever we want.

**Comment by yogeshnachnani** [comment]
i'll leave it for the assignee to merge and close this :)

File: runtime/src/main/java/com/flipkart/flux/config/FileLocator.java (hunk 1 above)
Comment (kartikaditya) [inline runtime/src/main/java/com/flipkart/flux/config/FileLocator.java]: This was my confusion too, when I read the code.
We cannot inject a static var, so should I be making deploymentRoot and thus FileLocator injectable?

File: runtime/src/test/java/com/flipkart/flux/controller/DashboardWebAppTest.java (hunk 2 above)
Comment (kartikaditya) [inline runtime/src/test/java/com/flipkart/flux/controller/DashboardWebAppTest.java]: Sure, will do

File: runtime/src/main/java/com/flipkart/flux/config/FileLocator.java (hunk 1 above)
Comment (yogeshnachnani) [inline runtime/src/main/java/com/flipkart/flux/config/FileLocator.java]: sure we can - https://github.com/google/guice/wiki/Injections

However, I agree we shouldn't. We can replace this with a decent builder or to constructor based injection. I don't see it as a pressing issue, but if you feel strongly about it then you can go ahead and do that - it'll be a trivial change

Comment (kartikaditya) [inline runtime/src/main/java/com/flipkart/flux/config/FileLocator.java]: I'l add a TODO to make it config based for now

**Pull request merged**